src/sentry/discover/endpoints/discover_key_transactions.py

```diff
@@ -110,7 +114,7 @@ def get(self, request, organization):
 
         queryset = KeyTransaction.objects.filter(organization=organization, owner=request.user)
         if not queryset.exists():
-            raise ResourceDoesNotExist
+            return Response({"data": []}, status=200)
```

Comment:
@wmak this may not work if there are multiple y-axis

Reply:
@dashed Ended up zerofilling the results cause that would be easier, let me know if that's an issue though and I can update.